docs/servo_power_test/index.css

```diff
@@ -163,32 +171,27 @@ body {
   width: 26%;
 }
 
-#shellScript p {
-  margin-top: 2px;
-  margin-bottom: 2px;
-}
-
 #d3area {
   width: 50%;
   overflow-y: scroll;
 }
 
-#console {
+#dut_console {
```

Comment:
(ditto) I think we should use kebab-case here for the ids.
https://google.github.io/styleguide/htmlcssguide.html#id_Attributes